backend/app/src/main/kotlin/io/tolgee/websocket/Types.kt

```diff
@@ -0,0 +1,7 @@
+package io.tolgee.websocket
+
+enum class Types() {
+  TRANSLATION_DATA_MODIFIED;
+
+  val typeName get() = name.lowercase()
```

Comment:
Nitpick here, but considering event names take the form of URI paths, I'd consider having the name dash-cased would be more consistent overall. Very minor though.